### DISCUSSION TITLE
Geojson integration

### DISCUSSION THREAD

**Discussion by BenOnTrack** [systemed/tilemaker#544] [Q&A] [answered]
Hi,

I'd like to know if there would be a way to integrate geojson into tilemaker. 

As of now I'm using `tippecanoe` to convert a geojson (containing Linestring based on highway = path) to a mbtiles, and then render it with maplibre.

The problem by doing so, it's that I find it hard to find accurate options for tippecanoe to generate a mbtiles that would fit the basemap mbtiles generated by tilemaker, knowing that tilemaker has all those simplify_below/level/ratio options. 

Basically, the goal is to make sure that any path (generated from the same .pbf source) from the geojson-mbtiles would match the same path from tilemaker-mbtiles. 

Unless a workaround would be to:
- convert geojson in .shp 
- ingest the .shp in the config file with the same options as the one used for the tilemaker-mbtiles

**Comment by systemed** [comment, marked as answer]
As yet it won’t do that. In a future version I’d like it to handle geojson in the same way it does shapefiles. For now your best bet is to use ogr2ogr to convert the geojson into a shapefile.

**Reply by BenOnTrack** [reply]
Thanks !
